quodlibet/quodlibet/browsers/playlists/util.py

```diff
@@ -46,6 +47,18 @@ def __init__(self, parent, playlist):
                              Gtk.ResponseType.YES)
 
 
+class ConfirmPlaylistDND(object):
+    def __init__(self, parent, source_playlist, target_playlist):
+        title = _('Add the songs from playlist "{A}" to playlist "{B}"?'
+            ).format(A=source_playlist, B=target_playlist)
+        description = ""
+        ok_text = _("_Add Songs")
+        self.runner = ConfirmationPrompt(parent, title, description, ok_text)
+
+    def run(self):
```

Comment:
Please add docstrings if there's a meaningful return result here (as there seems to be)